docs/tutorials/Martini3/Small_Molecule_Parametrization_FastForward/index.qmd

```diff
@@ -0,0 +1,264 @@
+---
+title: "Parametrization of a new small molecule using Fast_Forward"
+format: html
+---
+
+<hr>
+In case of issues, please contact [Chris Brasnett](c.s.brasnett@rug.nl) or open an [issue on the repo](https://github.com/Martini-Force-Field-Initiative/Martini-Force-Field-Initiative.github.io/issues).
+<hr>
+
+### Summary
+- [`Introduction`](#introduction)
+- [`Generate atomistic reference data`](#generate-atomistic-reference-data)
+- [`Atom-to-bead mapping`](#atom-to-bead-mapping)
+- [`Generate the CG mapped trajectory from the atomistic simulation`](#generate-the-cg-mapped-trajectory-from-the-atomistic-simulation)
+- [`Create the initial CG itp and tpr files`](#create-the-initial-cg-itp-and-tpr-files)
+- [`Generate target CG distributions from the CG mapped trajectory`](#generate-target-cg-distributions-from-the-cg-mapped-trajectory)
+    - [`On the choice of bonded terms for the CG model`](#on-the-choice-of-bonded-terms-for-the-cg-model)
+    - [`Index files and generation of target distributions`](#index-files-and-generation-of-target-distributions)
+- [`Create the CG simulation`](#create-the-cg-simulation)
+- [`Optimize CG bonded parameters`](#optimize-cg-bonded-parameters)
+- [`Comparison to experimental results, further refinements, and final considerations`](#comparison-to-experimental-results-further-refinements-and-final-considerations)
+   - [`Molecular volume and shape`](#molecular-volume-and-shape)
+- [`Tools and scripts used in this tutorial`](#tools-and-scripts-used-in-this-tutorial)
+- [`References`](#references)
+
+### Introduction
+<hr>
+
+In this tutorial, we will discuss how to build a Martini 3 topology for a new small molecule using a semi-automated method with the Fast_Forward Python package. This tutorial builds on ideas and techniques discussed in the [classic Small Molecule Parameterization tutorial](../Small_Molecule_Parametrization/index.qmd), and it is recommended that you do that tutorial before this one.
+
+
+As before, we will use as an example the molecule *1-ethylnaphthalene* (Fig. 1), and make use of `Gromacs` versions 2024.x or later. FILES AVAILABLE HERE
```

Comment:
this line will need to be edited by @danielpastor97 when he'll upload the files

Reply:
Sure thing @ricalessandri ! @csbrasnett please share with me the files when ready and I will add them to the cgmartini-library S3 bucket and update the link accordingly.